scss/_main.scss

```diff
@@ -78,32 +78,18 @@
 .container {
     display: grid;
     grid-template-rows: 1fr (repeat(6));
-    grid-template-columns: 1fr;
-    grid-template-areas:
-    "item1"
-    "item2"
-    "item3"
-    "item4"
-    "item5"
-    "item6";
+    grid-template-columns: repeat(1, 1fr);
```

Comment:
Just in general, so you are overwriting my stuff which will lead to merge conflicts, because I still work in my branch. All enhancements can be done later. Please revert the changes in this file that affects my work.